audbackend/core/backend/base.py

```diff
@@ -25,11 +26,15 @@ def __init__(
         self,
         host: str,
         repository: str,
+        *,
+        auth: typing.Any = None,
```

Comment:
Since we also say "repository" and not "repo", I would use the full word `authentication` here, too.

Reply:
Yes, good idea. I changed the `auth` argumet and attribute to `authentication`, and renamed the `authentication()` class method to `get_authentication()`.